provisioning_templates/provision/atomic_kickstart_default.erb

```diff
@@ -13,8 +13,11 @@ timezone --utc <%= host_param('time-zone') || 'UTC' %>
 <% else -%>
 <% dhcp = !@static -%>
 <% end -%>
+<% if subnet.respond_to?(:mtu) && !subnet.mtu.blank? -%>
```

Comment:
Since templates are now bramched per Foreman version I think the `respond_to?` check is unnecessary and should be removed.